### PR TITLE
PT-BR Translation

### PR DESCRIPTION
**Pull request by BowFreak**

### CODE REVIEW

File: assets/romfs/i18n/pt/gui.json [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+    "MOVE_4_ID": "Mover 4 ID",
+    "MOVE_4_PP_UPS": "Mover 4 PP Ups",
+    "MOVE_4_PP": "Mover 4 PP",
+    "MOVE_4": "Mover 4",
+    "MOVES": "Mover",
+    "MULTI_ABILITY_RIBBON": "Fita de Muitas Habilidades",
+    "NA": "N/A",
+    "NATIONAL_CHAMPION_RIBBON": "Fita do Campeão Nacional",
+    "NATIONAL_RIBBON": "Fita Nacional",
+    "NATIONALITY": "Nacionalidade",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/pt/gui.json]: Shorten this line

File: assets/romfs/i18n/pt/gui.json [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+    "CHAMPION_RIBBON": "Fita do Campeão",
+    "CHECKSUM": "CHECKSUM",
+    "CHN": "China",
+    "CIRCLE": "Círculo",
+    "CLASSIC_RIBBON": "Fita Clássica",
+    "CLEAR": "Limpo",
+    "CLEVERNESS_MASTER_RIBBON": "Fita do Mestre da Inteligência",
+    "CLONE": "Clone",
+    "CONFIG_BACKUP_INJECTION": "Habilitar injeção no backup",
+    "CONFIG_BACKUP_SAVE": "Automaticamente carregar ao abrir",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/pt/gui.json]: Shorten this line

File: assets/romfs/i18n/pt/gui.json [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+    "STORAGE_BAD_BALL": "Bola Impossível",
+    "STORAGE_BAD_FORM": "Forma Impossível",
+    "STORAGE_BAD_ITEM": "Item Impossível",
+    "STORAGE_BAD_MOVE": "Ataque Impossível",
+    "STORAGE_BAD_SPECIES": "Espécie Impossível",
+    "STORAGE_BAD_TRANFER": "Não foi possível transferir o Pok\u00E9mon:",
+    "STORAGE_CHECKBACK": "Cheque novamente daqui a uns anos.",
+    "STORAGE_IMPLEMENTATION": "Ainda não é possível fazer isso..",
+    "STORAGE_RESIZE": "Redimensionando armazenamento...",
+    "STORAGE": "Armazenamento",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+    "COOL_RIBBON_GREAT": "Grande Fita Legal",
+    "COOL_RIBBON_HYPER": "Híper Fita Legal",
+    "COOL_RIBBON_MASTER": "Fita Mestre Legal",
+    "COOL_RIBBON_SUPER": "Super Fita Legal",
+    "COOL_RIBBON_ULTRA": "Ultra Fita Legal",
+    "COOL_RIBBON": "Fita Legal",
+    "COOLNESS_MASTER_RIBBON": "Fita do Mestre em ser Legal",
+    "COUNTRY_ID": "ID da Região",
+    "COUNTRY_RIBBON": "Fita da Região",
+    "CTOT_FSHIP": "Amizade do CT/OT",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 5]
```diff
@@ -0,0 +1,501 @@
...
+    "SPDEF": "Defesa ESP.",
+    "SPECIAL_RIBBON": "Fita Especial",
+    "SPECIES": "Espécies",
+    "SPECIES_NAME": "Nome da Espécie",
+    "SPEED_EV": "EV de Velocidade",
+    "SPEED_IV": "IV de Velocidade",
+    "SPEED_LEVEL_1": "Nível de Velocidade 1",
+    "SPEED_LEVEL_2": "Nível de Velocidade 2",
+    "SPEED_LEVEL_3": "Nível de Velocidade 3",
+    "SPEED": "Velocidade",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 6]
```diff
@@ -0,0 +1,501 @@
...
+    "SMILE_RIBBON": "Fita do Sorriso",
+    "SNOOZE_RIBBON": "Fita da Soneca",
+    "SORT": "Por em ordem",
+    "SOUVENIR_RIBBON": "Fita de Souvenir",
+    "SPATK_EV": "EV de Ataque ESP.",
+    "SPATK_IV": "IV de Ataque ESP.",
+    "SPATK_LEVEL_1": "Nível de Ataque ESP. 1",
+    "SPATK_LEVEL_2": "Nível de Ataque ESP. 2",
+    "SPATK_LEVEL_3": "Nível de Ataque ESP. 3",
+    "SPATK.": "ATQ. ESP.",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 7]
```diff
@@ -0,0 +1,501 @@
...
+    "BANK_BACKUP": "Fazendo Backup do seu armazenamento...",
+    "BANK_BAD_CONVERT": "Não foi possível ler bank.bin",
+    "BANK_BOX_NAME": "Nome da BOX",
+    "BANK_CONFIRM_CLEAR": "Apagar a BOX seleciona?",
+    "BANK_CONFIRM_DUMP": "Fazer dump do Pok\u00E9mon selecionado?",
+    "BANK_CONFIRM_RELEASE": "Soltar o Pok\u00E9mon selecionado?",
+    "BANK_CONVERT": "Convertando bank....",
+    "BANK_CORRUPT": "Dados do bank estão corrompidos",
+    "BANK_CREATE": "Criando armazenamento...",
+    "BANK_FAILED_EXIT": "Não é permitido sair enquanto um Pok\u00E9mon está sendo segurado!",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 8]
```diff
@@ -0,0 +1,501 @@
...
+    "DOUBLE_ABILITY_RIBBON": "Fita de Habilidade Dupla",
+    "DOWNCAST_RIBBON": "Fita Abatida",
+    "DOWNLOADING_ASSETS": "Baixando banco de dados...",
+    "DRAG_DOWN_HYDREIGON": "Drag Down Hydreigon!",
+    "DUMP": "Dump",
+    "EARTH_RIBBON": "Fita da Terra",
+    "EDIT": "Editar",
+    "EDITOR_CHECK_EXIT": "Sair sem salvar?",
+    "EDITOR_CP": "CP: ",
+    "EDITOR_HIDDEN_POWER": "Hidden Power",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 9]
```diff
@@ -0,0 +1,501 @@
...
+    "GEOLOCATION_2": "Geolocalização 2",
+    "GEOLOCATION_3": "Geolocalização 3",
+    "GEOLOCATION_4": "Geolocalização 4",
+    "GEOLOCATION_5": "Geolocalização 5",
+    "GORGEOUS_RIBBON": "Fita Linda",
+    "GORGEOUS_ROYAL_RIBBON": "Fita Linda Real",
+    "GREAT_ABILITY_RIBBON": "Grande Fita de Habilidade",
+    "GREEN_RIBBON": "Fita Verde",
+    "HEART": "Coração",
+    "HELD_ITEM": "Item Segurado",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 10]
```diff
@@ -0,0 +1,501 @@
...
+    "TYPE2": "Tipo 2",
+    "UNKNOWN_FLAGS": "Bandeiras Desconhecidas",
+    "UNKNOWN": "Desconhecido",
+    "UNUSED": "Não usado",
+    "USA": "Estados Unidos",
+    "VICTORY_RIBBON": "Fita da Vitória",
+    "VIEW": "Ver",
+    "WATCH_OUT_THATS_ONE_TRICKY_SECOND_HALF": "Cuidado! Essa é uma segunda metade complicada!",
+    "WC_CHANGE_SLOT": "Aperte \uE000 para mudar o Slot",
+    "WC_DUMP1": "Aperte \uE000 para fazer dump no Wonder Card",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json [hunk 11]
```diff
@@ -0,0 +1,501 @@
...
+    "WC_DUMP2": "Aperte \uE002 para fazer dump no Wonder Card",
```
[the remark this answers is not in the document]
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json (hunk 1 above)
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

File: assets/romfs/i18n/pt/gui.json (hunk 2 above)
Comment (BowFreak) [reply assets/romfs/i18n/pt/gui.json]: finished it

**Pull request merged**